portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsEntries.testcase

```diff
@@ -143,13 +143,54 @@ definition {
 	}
 
 	@description = "Verify that with Single Approver Workflow enabled and after the user Submit for Publication, the form entry will have a pending label"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test WorkflowSubmissionChangesLabelToPending {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
+		
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Text");
+
+		FormsAdminNavigator.gotoSettings();
+
+		Form.configureWorkflow(workflowDefinition = "Single Approver");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.openPublishURL(publishFormURL = "${publishedFormPublicURL}");
+
+
+		FormFields.editText(
+			fieldName = "Text",
+			fieldValue = "Test Text");
+
+		FormPortlet.submitLocalizedSuccessfully(buttonName = "Submit for Publication");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoViewEntries();
+
+		FormsAdmin.viewFormTableBody(fieldBody = "Pending");
```

Comment:
@PauloVAS, can you please rename:

- `viewFormTableBody` to `viewFormEntryWorkflowStatus`
- `FORM_ENTRY_TABLE_BODY` to `FORM_ENTRY_WORKFLOW_STATUS`
- `key_fieldBody` to `key_formEntryWorkflowStatus`
- `fieldBody` to `formEntryWorkflowStatus`

This renaming will make your test use the same terms used on the Forms UI and code.

Thanks in advance. :slightly_smiling_face: